src/ucs/datastruct/ptr_map.inl

```diff
@@ -136,6 +136,18 @@ ucs_ptr_map_del(ucs_ptr_map_t *map, ucs_ptr_map_key_t key)
     return UCS_OK;
 }
 
+/**
+ * Test if key is indirect representation of value pointer.
+ *
+ * @param [in]  map     Container to get the pointer value from.
+ * @return non-zero if key is indirect representation, zero otherwise.
```

Comment:
can mention `1` instead of non-zero

Reply:
we already have "nonzero" above, also it can be compiler/c-standard specific since it's inline function we cannot guarantee `1`

Reply:
then why need `!!` ?

Reply:
to cast result of & to 4 byte integer correctly

Reply:
ic, ok